bcda/bcdacli/cli.go

```diff
@@ -362,6 +363,17 @@ func setUpApp() *cli.App {
 			Action: func(c *cli.Context) error {
 				ignoreSignals()
 				success, failure, skipped, err := cclf.ImportCCLFDirectory(filePath)
+				if err != nil {
+					log.API.Error("error returned from ImportCCLFDirectory: ", err)
+					cli.NewExitError("an error occurred during the cclf-import process, see logs for more details", 1)
+					return err
+				}
+				if failure > 0 || skipped > 0 {
+					log.API.Error("Successfully imported %v files.  Failed to import %v files.  Skipped %v files.  See logs for more details. ", err)
```

Comment:
@laurenkrugen-navapbc out of curiosity, how are `%v` vars populated here?

Reply:
I actually reverted the logging back to what it originally was, this time including the variables in the logging statement.